infra/modules/aws/main.tf

```diff
@@ -103,11 +103,6 @@ resource "aws_iam_policy" "execution_lambda_to_caller" {
   }
 }
 
-resource "aws_iam_role_policy_attachment" "invoker_lambda_execution" {
```

Comment:
Are you sure this works? Do lambdas log because they have the rights to do so, or because the caller (that can do InvokeURL) also has logging permissions?
Also this policy is mentioned in various doc files, see https://github.com/search?q=repo%3AWorklytics%2Fpsoxy%20AWSLambdaBasicExecutionRole&type=code . Does it need to be dropped in certain cases?

Reply:
lambdas don't execute as this role

they each have their own execution role; and the lambda execution permission is attached to each here:
https://github.com/Worklytics/psoxy/blob/02b8dc5de0951b0cec3695634fdfa0705851ddb1/infra/modules/aws-psoxy-lambda/main.tf#L140

Reply:
got it. Looks good then